front/app/containers/Admin/sideBar/NotificationsPopup.tsx

```diff
@@ -93,18 +93,18 @@ export const NotificationsPopup = () => {
                   fontSize="base"
                   textAlign="left"
                   my="0px"
-                  w="100%"
                 >
                   {formatMessage({ ...messages.notifications })}
                 </Text>
               </>
             )}
-            <Box
-              w="auto"
-              h={isSmallerThanPhone ? '0px' : '18px'}
-              ref={iconDivRef}
-            >
-              {unreadNotificationsCount > 0 && (
+            {!isSmallerThanPhone && unreadNotificationsCount > 0 && (
```

Comment:
Not showing the number of notifications when we only see the icons because that's what we do for the other item(s) at the moment, so for consistency.